src/Microsoft.DotNet.Wpf/src/ReachFramework/Serialization/VisualSerializer.cs

```diff
@@ -2026,27 +2026,9 @@ out bitmapToDrawingTransform
 
         private static bool EmbeddingAllowed(GlyphTypeface typeface)
         {
-            CodeAccessPermission fontReadPermission = typeface.CriticalFileReadPermission;
-
             FontEmbeddingRight embeddingRights = FontEmbeddingRight.Installable;
 
-
-            if (fontReadPermission != null)
-            {
-                fontReadPermission.Assert(); // Blessed assert
-            }
-
-            try
-            {
-                embeddingRights = typeface.EmbeddingRights;
-            }
-            finally
-            {
-                if (fontReadPermission != null)
-                {
-                    CodeAccessPermission.RevertAssert();
-                }
-            }
+            embeddingRights = typeface.EmbeddingRights;
```

Comment:
fix this method up a bit, it could just be one line

what were the previous expectations of this function and behavior? so do we operate under the assumption of full trust?

Reply:
there are a few methods after this that follow a similar pattern, this comment applies to all of those as well

Reply:
Generally, everything is now just a full trust path.  So all updates should make those assumptions.

Reply:
Updated this and the following methods to be one line. Now we just want a full trust path (which the one line update achieves).